frontend/src/util/LiveMessageManager.js

```diff
@@ -9,19 +10,24 @@ export const LIVE_MESSAGE_STATUS = {
   HEARTBEAT_FAILED: 'heartbeat_failed'
 }
 
+// increment this number each time the channel protocol is changed in an incompatible way
```

Comment:
This comment does not follow the pattern:
// INFO - RJ - 2020-08-12  - increment this number each time the channel protocol is changed in an incompatible way